guides/Tong/README.md

```diff
@@ -0,0 +1,173 @@
+## Written by Sharon Tong
```

Comment:
You may not see it yet but there is a thing called front matter which jekyll uses and it will need to be in your guide for the site to work. Use this on your first three lines.

```
---
layout: post
title: CentOS E-Commerce
author: Varghese Emlin Charly
---
```